mautic/mautic
# Add a local env file so DDEV runs in dev mode with debug enabled

…ays starts as dev environment with debug mode enabled, when running locally (or in gitpod)



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ Fixes #13108]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

With this change, a `.env.local` file is created when Mautic is installed, that way if running in DDEV (for local development or testing in Gitpod), debug bar is enabled and you are in dev mode



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. You should see the debug bar at the bottom of the window, or a square with the Symfony logo (if it is collapsed) which you can click to open the debug bar
